Add deployment button for Hostinger in README

Minijus-Sa opened the pull request:
I've added Hostinger 1-click installation badge for fast and easy intall. You can change the URL to contain the referral code to earn commission on every installation. You can also read more here: https://www.hostinger.com/support/deploy-on-hostinger-button/

Minijus-Sa committed:
Add deployment button for Hostinger in README

CooperDActor-bytes reviewed (changes requested):
the URL needs to be changed to https://raw.githubusercontent.com/Peppermint-Lab/peppermint/refs/heads/main/docker-compose.yml

Minijus-Sa committed:
Fix Hostinger deploy link in README

Updated the link in the Hostinger deployment badge to point to the main docker-compose.yml file.
